spec/i18n_spec.rb

```diff
@@ -24,6 +24,14 @@ def expectation(binding, inspection)
 
   context 'es locale' do
     before { I18n.locale = :es }
+
+    describe 'source exectations' do
+      it { expect(expectation('*', 'SourceRepeats:foo(X)').translate).to eq('la solución debe usar <strong>foo(X)</strong> más de una vez') }
+      it { expect(expectation('*', 'SourceContains:foo(X)').translate).to eq('la solución debe usar <strong>foo(X)</strong>') }
+      it { expect(expectation('*', 'SourceEquals:foo(X)').translate).to eq('la solución debe ser igual a <strong>foo(X)</strong>') }
```

Comment:
I'm not sure I get this inspection :thinking: wouldn't this just be giving the student the exercises solution? Is that something we want?

Reply:
ok, after reading the other PR I think I got it. The point of this inspection is the negated version, right? to check that something does **not** equal something else.

Reply:
Exactly